test/pilosa_test.go

```diff
@@ -0,0 +1,49 @@
+package test_test
+
+import (
+	"net/http"
+	"testing"
+
+	"encoding/json"
+
+	"github.com/pilosa/pilosa/test"
+)
+
+func TestNewCluster(t *testing.T) {
+	cluster := test.MustNewServerCluster(t, 3)
+	response, err := http.Get("http://" + cluster.Servers[0].Server.Addr().String() + "/status")
+	if err != nil {
+		t.Fatalf("getting schema: %v", err)
+	}
+	dec := json.NewDecoder(response.Body)
+	a := StatusResp{}
+	err = dec.Decode(&a)
```

Comment:
`err = `json.Unmarshal(response.Body, &a)` would make the code 1 line shorter by getting rid of `json.NewDecoder` line

Reply:
Unmarshal takes a byte slice but response.Body is an io.Reader. As a general rule, I like to stay in the habit of using Readers and Writers (and the APIs that support them) since they give you streaming capabilities.

Reply:
That makes sense.